src/libs/elektra/keytest.c

```diff
@@ -419,68 +449,68 @@ succeed_if (keyRel (key, check) >= 0, "should be the same or below");
  *
  * - If the keys are in the same hierarchy, a value smaller then -2 is returned.
  * It means that the key is not below.
-@verbatim
-user/key/myself
-user/key/sibling
-@endverbatim
+ @verbatim
+ user/key/myself
+ user/key/sibling
+ @endverbatim
  *
  * @code
-keySetName (key, "user/key/folder");
+ keySetName (key, "user/key/folder");
 keySetName (check, "user/notsame/folder");
 succeed_if (keyRel (key, check) < -2, "key is not below, but same namespace");
- * @endcode
```

Comment:
i used scripts/reformate-source which seems to call clang-format-3.8 + i have clang-format-3.8 installed.
just ran the script again, same issue